test/tap/groups/mariadb10-galera/pre-proxysql.bash

```
@@ -0,0 +1,61 @@
+# Pre-proxysql hook for mariadb10-galera group.
+# Runs on the HOST (not inside Docker). Uses docker exec to reach ProxySQL.
+# The infra (infra-dbdeployer-mariadb10-galera) is started by the framework AFTER this hook.
+# This hook configures ProxySQL with Galera-specific fallback hostgroups, users, and routing.
+#
+
+PROXY_CONTAINER="proxysql.${INFRA_ID}"
+INFRA="${DEFAULT_MYSQL_INFRA:-infra-dbdeployer-mariadb10-galera}"
+MYSQL_HOST="${MYSQL_PRIMARY_HOST:-dbdeployer1}.${INFRA}"
+
+run_admin() {
+    docker exec ${PROXY_CONTAINER} mysql -uradmin -pradmin -h127.0.0.1 -P6032 -e "$1" 2>&1 | grep -vP 'mysql: .?Warning' || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)